docs/advanced-config.md

```diff
@@ -0,0 +1,104 @@
+```
+`GIT_HOSTNAME` is only necessary if you're using Github Enterprise. Additional config and install information for Vencax's Oauth provider is available in [its repository](https://github.com/vencax/netlify-cms-github-oauth-provider).
+
+## Configure Netlify CMS
+Two files are required to add Netlify CMS to a static site, `admin/index.html` and `admin/config.yml`. Look at the [quick-start](https://www.netlifycms.org/docs/quick-start/) section for more info on where these go.
+
+```
+admin
+ ├ index.html
+ └ config.yml
+```
+
+### Index.html
+The index file can be rather simple, but if you're planning on using your own Oauth server, ensure you're getting at least v0.4.3 of cms.js by using unpkg.com. Non-Netlify Oauth servers will not work with prior versions.
+``` html
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Content Manager</title>
+
+  <link rel="stylesheet" href="https://unpkg.com/netlify-cms@~0.4.3/dist/cms.css" />
+
+</head>
+<body>
+  <script src="https://unpkg.com/netlify-cms@~0.4.3/dist/cms.js"></script>
+</body>
+</html>
+```
+
+### Config.yml
+
+The `config.yml` file will vary wildly from site to site, so we'll go section by section.
+
+#### Backend
+Here is a more complex backend configuration example. Each item will be explained below:
+```yaml
+backend:
+  # REQUIRED CONFIG
+  name: github
+  repo: user/repository
+  branch: master
+  # OPTIONAL CONFIG. No trailing slashes on URLs
+  api_root: https://github.some.domain.com/api/v3
+  site_id: static.site.url.com
+  base_url: https://auth.server.url.com
+```
+
+* **name** is almost always `github` unless you're using `netlify-auth` or `test-repo`, which won't be covered here. It determines which type of api you're using.
+* **repo** tells the CMS what repo to push to.
+* **branch** tells the CMS which branch to push to.
+* **api_root (optional)** is only necessary if you're hosting your repo on a Github Enterprise account. It's the api endpoint.
+* **site_id (optional)** is necessary if you want to test on your local machine, and allows CORS between the api and the CMS.
+* **base_url (optional)** tells the CMS where the Oauth server is. This is _required_ when using Vencax's Oauth provider.
+
+Another convenient feature is the ability to have a different backend configuration depending on if you're running a development or production environment. This can be done like so:
+
+```yaml
+backend:
+  name: netlify-auth
+  repo: user/repository
+  branch: dev
+
+production:
+  name: github
+  repo: user/repository
+  branch: master
+  api_root: https://github.some.domain.com/api/v3
+  site_id: static.site.url.com
+  base_url: https://auth.server.url.com
+```
+
+#### Collections
+You can find info on collections and translating your static site yaml front-matter to cms fields in the [quick-start](https://www.netlifycms.org/docs/quick-start/) section.
+
+### Continuous Delivery
```

Comment:
Should have been more clear - I meant the Continuous Delivery section is out of scope, not CMS related.